<launch>

    <!-- Enable custom logger configuration-->
    <env name="ROS_PYTHON_LOG_CONFIG_FILE" value="$(find mapc_rhbp_ettlinger)/launch/python_logging.conf"/>

    <!--Infrastructure Nodes-->
    <include file="$(find mapc_rhbp_ettlinger)/launch/graphhopper.launch">
        <arg name="initial_port" value="8989" />
    </include>

    <!--Planner Nodes-->

    <include file="$(find mapc_rhbp_ettlinger)/launch/eamanager.launch" pass_all_args="true">
    </include>

    <include file="$(find mapc_rhbp_ettlinger)/launch/planner.launch" pass_all_args="true">
    </include>

    <include file="$(find mapc_rhbp_ettlinger)/launch/debug_agent.launch" pass_all_args="true">
    </include>


    <!-- Drones -->
        <!-- Bridges -->
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI1" />
        <arg name="only_agent_specific" value="False" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>

    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI2" />
        <arg name="only_agent_specific" value="False" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>

    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI3" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>

    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI4" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI5" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI6" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI7" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI8" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI9" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI10" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI11" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI12" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI13" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI14" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI15" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI16" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI17" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI18" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI19" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI20" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI21" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI22" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI23" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI24" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI25" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI26" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI27" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI28" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI29" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI30" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI31" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI32" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI33" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>
    <include file="$(find mac_ros_bridge)/launch/mac_ros_bridge.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI34" />
        <arg name="server_ip" value="agentcontest2.in.tu-clausthal.de" />
        <arg name="server_port" value="12300" />
        <arg name="password" value="xp8Wqm" />
    </include>

    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI1" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI2" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI3" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI4" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI5" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI6" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI7" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI8" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI9" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI10" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI11" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI12" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI13" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI14" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI15" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI16" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI17" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI18" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI19" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI20" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI21" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI22" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI23" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI24" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI25" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI26" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI27" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI28" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI29" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI30" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI31" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI32" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI33" />
    </include>
    <include file="$(find mapc_rhbp_ettlinger)/launch/rhbp_agent.launch" pass_all_args="true">
        <arg name="agent_name" value="TUBDAI34" />
    </include>



    <param name="situationBias" type="double" value="1.0" />
    <param name="predecessorBias" type="double" value="1.0" />
    <param name="successorBias" type="double" value="1.0" />
    <param name="conflictorBias" type="double" value="1.0" />
    <param name="goalBias" type="double" value="1.0" />
    <param name="planBias" type="double" value="1.0" />
    <param name="activationThreshold" type="double" value="7.0" />
    <param name="createLogFiles" type="bool" value="false"/>
    <param name="activation_algorithm" type="string" value="default"/>
    <param name="max_parallel_behaviours" type="int" value="1"/> <!--Comment or negative for unlimited-->

    <param name="RhbpAgent.MAX_DECISION_MAKING_TIME" type="double" value="7"/>
    <param name="RhbpAgent.BUILD_WELL_ENABLED" type="bool" value="False"/>


    <param name="ShouldBidForAssemblyDecision.WEIGHT_LOAD" type="double" value="30"/>
    <param name="ShouldBidForAssemblyDecision.WEIGHT_INGREDIENT_LOAD" type="double" value="100"/>
    <param name="ShouldBidForAssemblyDecision.WEIGHT_STEPS" type="double" value="-3"/>
    <param name="ShouldBidForAssemblyDecision.ACTIVATION_THRESHOLD" type="double" value="-15"/>

    <param name="ChooseStorageForHoardingDecision.WEIGHT_STEPS"  value="-1" type="double"/>
    <param name="ChooseStorageForHoardingDecision.WEIGHT_ITEMS_ALREADY_THERE"  value="-1" type="double"/>

    <param name="ChooseItemToGatherMechanism.WEIGHT_STEPS"  value="1" type="double"/>
    <param name="ChooseItemToGatherMechanism.WEIGHT_ASSEMBLY_ROLE_MATCH"  value="2.5" type="double"/>
    <param name="ChooseItemToGatherMechanism.WEIGHT_ASSEMBLY_ROLE_MATCH_COUNT"  value="0.5" type="double"/>
    <param name="ChooseItemToGatherMechanism.WEIGHT_PRIORITY"  value="200" type="double"/>
    <param name="ChooseItemToGatherMechanism.FINISHED_PRODUCT_PRIORITY_TO_INGREDIENT_CONVERSION"  value="3" type="double"/>


    <param name="ChooseBestAvailableJobDecision.PERCNTILE_TO_TRY_JOB"  value="0.7" type="double"/>
    <param name="ChooseBestAvailableJobDecision.BID_PERCENTILE"  value="50" type="int"/>
    <param name="ChooseBestAvailableJobDecision.TIME_LEFT_WEIGHT_START"  value="30" type="int"/>
    <param name="ChooseBestAvailableJobDecision.ACTIVATION_THRESHOLD"  value="-50" type="double"/>
    <param name="ChooseBestAvailableJobDecision.IMPORTANT_JOB_PERCENTILE"  value="0.95" type="double"/>
    <param name="ChooseBestAvailableJobDecision.ACTIVATION_TO_DESIRED_PRODUCT_CONVERSION"  value="0.2" type="double"/>
    <param name="ChooseBestAvailableJobDecision.WEIGHT_PERCENTILE"  value="10.0" type="double"/>
    <param name="ChooseBestAvailableJobDecision.WEIGHT_TIME_PASSED"  value="-0.3" type="double"/>
    <param name="ChooseBestAvailableJobDecision.WEIGHT_TIME_OVER"  value="-0.4" type="double"/>
    <param name="ChooseBestAvailableJobDecision.PRIORITISE_MISSION_JOBS"  value="true" type="bool"/>
    <param name="ChooseBestAvailableJobDecision.WEIGHT_MISSION_JOB_PRIORITY"  value="2.0" type="double"/>


    <param name="ChooseBestJobAgentCombinationDecision.MIN_STEP_BUFFER"  value="5" type="double"/>
    <param name="ChooseBestJobAgentCombinationDecision.WEIGHT_LOAD"  value="8" type="double"/>
    <param name="ChooseBestJobAgentCombinationDecision.WEIGHT_INGREDIENT_LOAD"  value="8" type="double"/>
    <param name="ChooseBestJobAgentCombinationDecision.WEIGHT_NO_STORAGE_NEEDED"  value="20" type="double"/>
    <param name="ChooseBestJobAgentCombinationDecision.WEIGHT_STEPS"  value="-1" type="double"/>
    <param name="ChooseBestJobAgentCombinationDecision.PRIORITY_ACTIVATION_THRESHOLD"  value="-9000" type="double"/>
    <param name="ChooseBestJobAgentCombinationDecision.ACTIVATION_THRESHOLD"  value="-30000" type="double"/>

    <param name="ContractNetManager.DEADLINE_BIDS"  value="2" type="double"/>
    <param name="ContractNetManager.DEADLINE_ACKNOWLEDGEMENT"  value="2" type="double"/>

    <param name="MainAssembleAgentDecision.WEIGHT_FINISHED_PRODUCT_FACTOR"  value="-1" type="double"/>
    <param name="MainAssembleAgentDecision.WEIGHT_BID_SKILL"  value="-100" type="double"/>
    <param name="MainAssembleAgentDecision.WEIGHT_BID_SPEED"  value="100" type="double"/>

    <param name="BestTUBDAIssemblyCombinationDecision.WEIGHT_NUMBER_OF_AGENTS" type="double" value="-0.2"/>
    <param name="BestTUBDAIssemblyCombinationDecision.MAX_NR_OF_AGENTS_TO_CONSIDER" type="int" value="17"/>
    <param name="BestTUBDAIssemblyCombinationDecision.MAX_PRIORITY_NOT_NEEDED_ITEMS" type="double" value="0.30"/>
    <param name="BestTUBDAIssemblyCombinationDecision.MAX_COUNT_NOT_NEEDED_ITEMS" type="int" value="7"/>

    <param name="BestTUBDAIssemblyCombinationDecision.MAX_AGENTS" type="int" value="7"/>
    <param name="BestTUBDAIssemblyCombinationDecision.MIN_AGENTS" type="int" value="3"/>
    <param name="BestTUBDAIssemblyCombinationDecision.MAX_STEPS" type="int" value="20"/>
</launch>